lib/src/checkers/native_host_reachability_checker.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:io';
+
+import 'package:connecteo/connecteo.dart';
+import 'package:connecteo/src/connection_entry.dart';
+import 'package:connecteo/src/host_reachability_checker.dart';
+
+final _defaultAddresses = List<ConnectionEntry>.unmodifiable([
```

Comment:
This is mostly suggestion but with newest dart we can create enums with values or sealed classes,

Maybe this will be a more nice way to declare ConnectionEntry

Reply:
We have 2 path
1) When user don't use any additional addreses we use defaultAdresses list
2) When user will add own addresses we replace defaultAdresses with list from user.

So I think it is easier for user use list instead of create enum.

`_connecteo = ConnectionChecker(
      checkAddresses: [
        ConnectionEntry(
          'one.one.one.one',
          ConnectionEntryType.apiUrl,
        )
      ],`